c_api/Clustering_c.h

```diff
@@ -75,14 +75,20 @@ FAISS_DECLARE_GETTER(Clustering, size_t, d)
 /// getter for k
 FAISS_DECLARE_GETTER(Clustering, size_t, k)
 
+FAISS_DECLARE_CLASS(ClusteringIterationStats)
```

Comment:
While these lines will correctly declare the class and its getters, they won't provide an implementation. Attempting to call any of these methods would fail at link time. Please head over to Clustering_c.cpp and add them through the `DEFINE_GETTER` macro (should be easy to follow by example).

Reply:
done thanks !!
https://github.com/facebookresearch/faiss/pull/1140/commits/3f9b817c455ebbd2bdf27fed1a1f0e20416bf072